### ISSUE TITLE
speak typed characters and words not working in google groups message composer

### ISSUE THREAD

**Issue by vortex1024** [nvaccess/nvda#7957]
### Steps to reproduce:
1. open a google group page in firefox.
2. compose a new message.
3. Turn speak typed characters off and speak typed words on.
4. Type some text.
### Expected behavior:
NVDA shouldn't speak each individual character and speak each word when the space key is pressed.

### Actual behavior:
NVDA does speak each character, ignoring the speak typed character setting, and doesn't speak the word when space is pressed, ignoring the speak typed words setting.

### System configuration:
NVDA version: 2017.4

NVDA Installed or portable: installed

Windows version: 10.1709

Name and version of other software in use when reproducing the issue: firefox 56

### Other questions:

Does the issue still occur after restarting your PC? yes

**Comment by ehollig** [comment]
I believe this has something to do with live regions, but it would be good if someone else could confirm this.

**Comment by derekriemer** [comment]
Yes, this is a live region announcing each and every character you type. Quite unnecessary and annoying.

**Comment by PratikP1** [comment]
I’m reporting this to Google’s general accessibility discussion list as something to fix. If google implements braille support as they have in Docs and other Google Suite products, this issue will be fixed after #7935 is fully incorporated into NVDA.
